<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVisualLogDialog</class>
 <widget class="QWidget" name="CVisualLogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>175</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,1">
     <item>
      <widget class="QFrame" name="frame">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QToolButton" name="VLB_FIRSTFRAME">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>First Frame</string>
          </property>
          <property name="icon">
           <iconset resource="VisualLogControls.qrc">
            <normaloff>:/dialog/VisualLogControls/playerfirst.png</normaloff>
            <disabledoff>:/dialog/VisualLogControls/playerfirst_disabled.png</disabledoff>:/dialog/VisualLogControls/playerfirst.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="VLB_PREVFRAME">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Previous Frame</string>
          </property>
          <property name="icon">
           <iconset resource="VisualLogControls.qrc">
            <normaloff>:/dialog/VisualLogControls/playerprev.png</normaloff>
            <disabledoff>:/dialog/VisualLogControls/playerprev_disabled.png</disabledoff>:/dialog/VisualLogControls/playerprev.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="VLB_PLAY">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
          <property name="icon">
           <iconset resource="VisualLogControls.qrc">
            <normaloff>:/dialog/VisualLogControls/playerplay.png</normaloff>
            <disabledoff>:/dialog/VisualLogControls/playerplay_disabled.png</disabledoff>:/dialog/VisualLogControls/playerplay.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="VLB_STOP">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="VisualLogControls.qrc">
            <normaloff>:/dialog/VisualLogControls/playerstop.png</normaloff>
            <disabledoff>:/dialog/VisualLogControls/playerstop_disabled.png</disabledoff>:/dialog/VisualLogControls/playerstop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="VLB_NEXTFRAME">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Next Frame</string>
          </property>
          <property name="icon">
           <iconset resource="VisualLogControls.qrc">
            <normaloff>:/dialog/VisualLogControls/playernext.png</normaloff>
            <disabledoff>:/dialog/VisualLogControls/playernext_disabled.png</disabledoff>:/dialog/VisualLogControls/playernext.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="VLB_LASTFRAME">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Last Frame</string>
          </property>
          <property name="icon">
           <iconset resource="VisualLogControls.qrc">
            <normaloff>:/dialog/VisualLogControls/playerlast.png</normaloff>
            <disabledoff>:/dialog/VisualLogControls/playerlast_disabled.png</disabledoff>:/dialog/VisualLogControls/playerlast.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="VLB_FOLDER">
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="VLEDIT_FOLDERPATH">
          <property name="text">
           <string>not set</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="8,1,0">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="VLSTATIC_SPEED">
              <property name="text">
               <string>0.00</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>fps</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Backwards</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>|</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Forward</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Playback speed:</string>
         </property>
         <property name="buddy">
          <cstring>VLSLIDER_SPEED</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSlider" name="VLSLIDER_SPEED">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1,1">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="VLSTATIC_FRAME">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QCheckBox" name="VLCHK_UPDATETEXT">
           <property name="text">
            <string>Update text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="VLCHK_UPDATEIMAGES">
           <property name="text">
            <string>Update images</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="VLCHK_KEEPASPECTR">
           <property name="text">
            <string>Keep aspect ratio</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="VLB_BACKGROUND">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>106</red>
               <green>104</green>
               <blue>100</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Background</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Current frame:</string>
       </property>
       <property name="buddy">
        <cstring>VLSLIDER_CURFRAME</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="VLSLIDER_CURFRAME">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="VisualLogControls.qrc"/>
 </resources>
 <connections/>
</ui>
